pepper/order_transform/generate.py

```diff
@@ -30,9 +31,12 @@ def hash_pedersen(left, right):
 	H = Point.from_x(FQ(16540640123574156134436876038791482806971768689494387082833631921987005038935))
 	return G * left + H * right
 
+def transform_int128(hash):
```

Comment:
Can you add a comment here describing what the method does? It's using some fancy python logic. It might also be slightly more explicit to return a pair (left, right) and print that instead.